frontend/src/components/shared/TooltipIcon/styled-components.ts

```diff
@@ -16,17 +16,38 @@
 
 import styled from "@emotion/styled"
 
-export const StyledTooltipIconWrapper = styled.div(({ theme }) => ({
-  display: "flex",
-  alignItems: "center",
+interface StyledTooltipIconWrapperProps {
+  isLatex?: boolean
+}
+
+export const StyledTooltipIconWrapper =
+  styled.div<StyledTooltipIconWrapperProps>(({ isLatex, theme }) => ({
+    display: "flex",
+    alignItems: "center",
+    marginTop: isLatex ? "12px" : "0",
 
-  svg: {
-    stroke: theme.colors.fadedText40,
-    strokeWidth: 2.25,
-  },
-  ":hover": {
     svg: {
       stroke: theme.colors.fadedText60,
+      strokeWidth: 2.25,
     },
-  },
+    ":hover": {
+      svg: {
+        stroke: theme.colors.fadedText60,
+      },
+    },
```

Comment:
Could you confirm intended styling behavior here? 
Looks like the stroke used here on hover in line 35 is the same theme color as in line 30 regularly. Previously it looks like we had a mild distinction on hover (theme.colors.fadedText40 to theme.colors.fadedText60).

Reply:
Yes, it was intended I will contact you privately on this

Reply:
Nit: since this is confirmed design decision to not alter coloration on hover, we can remove lines 33-37